components/eam/src/chemistry/modal_aero/aero_model.F90

```diff
@@ -1654,7 +1666,7 @@ subroutine modal_aero_bcscavcoef_init
     ! set up temperature-pressure pair to compute impaction scavenging rates
     real(r8), parameter :: temp_0C = 273.16_r8        ! K
     real(r8), parameter :: press_750hPa = 0.75e6_r8   ! dynes/cm2
-#include "../yaml/aero_model/f90_yaml/modal_aero_bcscavcoef_init_beg_yml.f90"
+!#include "../yaml/aero_model/f90_yaml/modal_aero_bcscavcoef_init_beg_yml.f90"
```

Comment:
I think these are commented out as you have generated data for these previously. You can uncomment these if you like.

Reply:
Thank you for catching that! I will uncomment them.